docs/tutorials/hello_many_worlds.ipynb

```diff
@@ -915,6 +917,72 @@
         "From this plot you can see that the neural network has learned to overcome the systematic miscalibration."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "C2RfWismj66S",
+        "colab_type": "text"
+      },
+      "source": [
+        "### 2.6 Verifying outputs\n",
+        "Now that you have trained this model, you can use it to correct your system that has this calibration error. Using Cirq you can do this with:"
```

Comment:
```suggestion
        "Now use the trained model to correct the qubit calibration errors. With Cirq:"
```

changed "system" to "qubit" to match the opening statement

Reply:
I like that a lot better, thanks!